torch/fx/experimental/proxy_tensor.py

```diff
@@ -145,6 +145,10 @@ def proxy_call(proxy_mode, func_overload, args, kwargs=None):
     func = func_overload.overloadpacket
     if func_overload in CURRENT_DECOMPOSITION_TABLE:
         return CURRENT_DECOMPOSITION_TABLE[func_overload](*args, **kwargs)
+    with proxy_mode.restore():
```

Comment:
Under which context would this see CompositeImplicit functions today?
Is it mostly linked to the functionalization use case?

Reply:
It also affects calls to torch.* inside decompositions, see the reentrant test added.